proto/public/api.proto

```diff
@@ -175,6 +175,8 @@ message StatTable {
 
       uint64 meshed_pod_count = 3;
       uint64 total_pod_count = 4;
+      uint64 errored_conduit_pod_count = 6;
```

Comment:
Can you add a comment here for this new field indicating how these are counted?  Particularly, i'm curious if we are sure these are conduit-specific errors or just any error running a pod?

Reply:
added! it's any error such that the pod is `PodFailed`.  the pod has to be a conduit controller pod to be counted.

this field is only useful if we're querying resources that have the conduit controller pods in them (e.g. querying all namespaces or querying namespace conduit from the ServiceMesh page)

Reply:
Yeah, that's probably better. The use case is maybe a little to specific as is.